### ISSUE TITLE
Missing alt_ids

### ISSUE THREAD

**Issue by drseb** [obophenotype/human-phenotype-ontology#4027]
- [x] Class obsolete Abnormality of globe location or size (HP:0000489) does not have a proper replacement via alt_ids!
- [x] Class obsolete Aplasia/Hypoplasia of the choroid (HP:0001122) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Increased prevalence of valvular disease (HP:0001726) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormal cardiological findings (HP:0003114) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Impaired FSH and LH secretion (HP:0003295) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormal dermatological laboratory findings (HP:0003340) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Peripheral retinal pigmentation abnormalities (HP:0200099) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Loss of heterozygosity, multiple chromosomes (HP:0003494) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Anaphylactoid purpura (HP:0200144) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Laterally deviated thumb phalanges (HP:0004066) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Laterally deviated terminal thumb phalanx (HP:0004083) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Advanced maturation/advanced ossification of terminal thumb phalanx epiphysis (HP:0004090) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radially deviated index finger phalanges (HP:0004110) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radially displaced proximal index finger phalanx (HP:0004121) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Metaphyseal abnormality of middle phalanx of the 2nd finger (HP:0004138) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Flared metaphysis of middle phalanx of index finger (HP:0004139) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radially deviated terminal index finger phalanx (HP:0004143) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Overgrowth of middle finger (HP:0004153) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Periosteal new bone of middle finger phalanges (HP:0004161) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radially pointed middle finger phalanges (HP:0004162) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radially pointed proximal middle-finger phalanx (HP:0004168) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Periosteal new bone of middle phalanx of middle-finger (HP:0004175) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormality of the epiphyses of the terminal phalanx of the middle finger (HP:0004183) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Cone-shaped epiphysis of terminal phalanx of the middle finger (HP:0004184) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Fused epiphysis of terminal phalanx of the middle finger (HP:0004185) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Large epiphysis of terminal phalanx of the middle finger (HP:0004186) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Prematurely fused epiphysis of terminal phalanx of the middle finger (HP:0004187) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Expanded phalanges of the ring finger (HP:0004193) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Hypoplastic phalanges of the ring finger (HP:0004194) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Short phalanges of the ring finger (HP:0004196) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Wide/broad phalanges of the ring finger (HP:0004198) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Severe T lymphocytopenia (HP:0005379) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Exaggerated cellular immune processes (HP:0005397) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Joint swelling onset late infancy (HP:0005833) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Joint dislocations in young adult (HP:0005837) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Corticospinal tract disease in lower limbs (HP:0007163) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Decreased retinal pigmentation with dispersion (HP:0007659) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Short curly eyelashes (HP:0007691) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Hypoplasia of choroid (HP:0007757) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Butterfly retinal pigment epithelial dystrophy (HP:0007783) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Fishnet retinal pigmentation (HP:0007801) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Progressive bifocal chorioretinal atrophy (HP:0007810) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Diffuse retinal cone degeneration (HP:0007829) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Temporal displacement of maculae (HP:0007851) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Pericentral pigmentary retinopathy (HP:0007852) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Small anterior lens surface opacities (HP:0007916) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Foveal hyperplasia (HP:0007923) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormal test result (HP:0500014) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Facial diplegic appearance (HP:0009090) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormal form of the 5th finger (HP:0009163) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Aplasia of the phalanges of the 3rd finger (HP:0009448) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Radial deviation of the thumb (HP:0009620) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Ulnar deviation of the thumb (HP:0009621) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Prenatal short stature (HP:0009885) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Transposition of the great arteries with intact ventricular septum (HP:0011606) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Transposition of the great arteries with ventricular septal defect (HP:0011607) does not have a proper replacement via alt_ids!
- [ ] Class obsolete Abnormality of common carotid artery plus branches (HP:3000026) does not have a proper replacement via alt_ids!

ping: @pnrobinson @nicolevasilevsky @LCCarmody

**Comment by drseb** [comment]
Ok. Most of them have a replaced_by or consider annotation. Will close and file a new ticket with the classes that have neither consider nor replaced_by!!

**Comment by cindyJax** [comment]
FYI- This issue appears to have broken the ontology load at MGI as well...

**Comment by drseb** [comment]
thanks @cindyJax . I hope MGI can load classes with replaced_by or consider tag. The other ones are definitely errors. @pnrobinson we need to decide on strategy for alt_ids here!

**Comment by pnrobinson** [comment]
Hi Seb,

your comments are good, we should do this. On vacation now and just briefliy at computer, I will work on this next week!


Peter Robinson
Professor and Donald A. Roux Chair, Genomics and Computational Biology
The Jackson Laboratory for Genomic Medicine
860.837.2095 t | peter.robinson@jax.org | https://robinsongroup.github.io/
Peter Robinson
________________________________

**Comment by drseb** [comment]
Before working on these, lets decide on a strategy!! I will now fix the obsolete classes that have neither replaced_by nor consider. Not sure what to do with the other classes. I will be on holiday next week, but available via email (I hope)

**Comment by pnrobinson** [comment]
I think our strategy was that if a class was merged with another, then it gets "replaced by", and if we thought a class was simply wrong, we chose the nearest term and called in "consider". We basically need to go through and examine these. I did this last year (I thought), not sure why there are still so many....

**Comment by drseb** [comment]
There aren't many that are really missing information. See https://github.com/obophenotype/human-phenotype-ontology/issues/4031
